### DISCUSSION TITLE
Need context for a string

### DISCUSSION THREAD

**Discussion by StoyanDimitrov** [arendst/Tasmota#13476] [Q&A] [answered]
Preparing a translation I stumbled on _Wifimanager set AccessPoint and keep Station_ string. ([`D_WIFIMANAGER_SET_ACCESSPOINT_AND_STATION`](https://github.com/arendst/Tasmota/blob/af85cbedb1353a1a0506077990046f4ea5527a7e/tasmota/xdrv_01_webserver.ino#L631)) Can I have more details about this string? What its purpose is and what it means in plain English so I can make meaningful translation.

**Comment by ascillato** [comment, marked as answer]
That string is just for the console logging.

All devices can be:
* in AP mode (access point mode) in which you see a new Wi-Fi network made by the device itself. That is used mainly to configure it.
* in STA mode (station mode) in which the device connects to a Wi-Fi router (the most used mode)
* AP+STA mode, in which a device is connected to a Wi-Fi router AND also it provides its own Wi-Fi network. This mode is used mainly when you are configuring the initial Wi-Fi credentials. So you are connected to a Tasmota AP with your phone and Tasmota keeps the AP with your phone while it is testing the credentials you have provided to a Router. So it is in dual mode (AP+STA mode)

So, that logging message is displayed on the Tasmota console ONLY when Tasmota is in DUAL mode: Access Point and Station
